package.json

```
@@ -87,6 +87,16 @@
           }
         ],
         "priority": "default"
+      },
+      {
+        "viewType": "one.editor.jsonTracer",
+        "displayName": "chromeTrace",
+        "selector": [
+          {
+            "filenamePattern": "*.trace.json"
+          }
+        ],
+        "priority": "default"
```

It would be better to open this json to be opened not in vscode explorer but in ONE explorer.

```suggestion
        "priority": "option"
```

---

Except `.cfg`, other file extensions are registered as `default`.
Should we change them all?

---

Sure. That would be awesome. I guess that could be done in next PR. :-)